src/assets/scripts/aircraft/AircraftController.js

```diff
@@ -359,6 +347,7 @@ export default class AircraftController {
         return this.aircraft.models[icao];
     }
 
+    // TODO: what is an `eid` and why would it beed to be updated?
```

Comment:
those circular dependencies are wicked.  eid, ahh, yes that makes sense.  So they aren't used for anything and can be removed?  I'll add an issue for that (#161).

One thing to remember with an id is that once it assigned it should _never_ change.  because then you need to know if the id you are looking for is updated or a previous version, etc.  That gets messy in a hurry, as I'm sure you can imagine.